README.md

```diff
@@ -79,6 +79,10 @@ apply(plugin= "org.jetbrains.dokka")
 ```
 
 ```kotlin
+dependencies {
+    dokkaHtmlPlugin("org.jetbrains.dokka:android-documentation-plugin:${dokka_version}")
+}
```

Comment:
Since not everyone needs this plugin and there's no explanation of what it's used for, I'd rather have it described separately down below, in the end of the `Android` section.

Something like

> In order to improve documentation experience on Android platform (such as support for `@hide` tag), consider adding the [android plugin](https://kotlin.github.io/dokka/1.6.10/user_guide/android-plugin/android-plugin/):
> ```
> dependencies {
>    dokkaHtmlPlugin("org.jetbrains.dokka:android-documentation-plugin:${dokka_version}")
> }
> ```

And the same `dokkaHtmlPlugin("org.jetbrains.dokka:android-documentation-plugin:${dokka_version}")` could be added to that page as well (website doc files are in the `docs` module).

What do you think about that?

Reply:
I'll respond to both of your comments here as they're pretty related.

I'd expect that the section on android has all of the configuration needed for an android project & that I'd just need to copy paste the example to get all of it.

My team found the plugin system pretty confusing. That might just be because we wanted to throw the config in gradle and not think about it that much, but I'd also argue that's how the vasty majority of people will be using the library. It could also be that whenever I see a `"org.jetbrains.dokka:android-documentation-plugin:${dokka_version}"` my lizard brain just immediately associates that with `implementation` commands etc, but IMHO that's another argument to make it more copy paste friendly.

After diving into the project more it did make sense.